packages/lesswrong/server/search/utils.ts

```diff
@@ -196,6 +196,7 @@ Posts.toAlgolia = async (post: DbPost): Promise<Array<AlgoliaPost>|null> => {
     curated: !!post.curatedDate,
     legacy: post.legacy,
     commentCount: post.commentCount,
+    // TODO: handle afCommentCount
```

Comment:
Currently we don't display commentCount on algolia results, and using commentCount works fine insofar as it's used for sorting results. It still seems like we should raise a flag whenever someone users post.commentCount.